src/opm/io/eclipse/EclFile.cpp

```diff
@@ -342,7 +342,7 @@ bool EclFile::is_ix() const
                 int p, first;
 
                 for (auto val : realStr) {
-                    double dtmpv = abs(std::stod(val));
+                    double dtmpv = fabs(std::stod(val));
```

Comment:
Please use `std::abs()` and include `<cmath>` instead.

Reply:
I disagree there: since abs() can silently end up calling abs(int), eat a double and return an int (if the stars and includes are wrong), I think fabs is better to use (in non-generic contexts as here). I think making abs() live in  two different places is a library design flaw.

Reply:
Well, if you forget to include cmath that might indeed happen. But using a C-function might not be the best solution here. If you have concerns about implicit (wrong) conversion simply use `std::fabs`, instead.

Reply:
> If you have concerns about implicit (wrong) conversion simply use `std::fabs`, instead.

Thanks for the correction. That is what I meant, just not what I wrote...